gitlab-runner.yaml

```diff
@@ -288,13 +292,11 @@ Resources:
                       sudo gitlab-runner register \
                       --non-interactive \
                       --url "${GitlabServerURL}" \
-                      --registration-token $GitlabGroupToken \
+                      --token $GitlabGroupToken \
                       --executor "docker" \
                       --docker-image "${DockerImagePath}" \
                       --description "Gitlab Runner with Docker Executor" \
-                      --locked="${isLOCKED}" --access-level "${ACCESS}" \
-                      --docker-volumes "/var/run/docker.sock:/var/run/docker.sock" \
-                      --tag-list "${RunnerEnvironment}-${RunnerVersion}-docker"
```

Comment:
Do you know if there a new way to tag the runners?

Reply:
In the new Gitlab runner registration token workflow, sensitive settings should be determined by an administrator at the time of new gitlab runner project creating. the tag also includes this change and will be defined when registering the token in  UI.

![image](https://github.com/aws-samples/amazon-ec2-gitlab-runner/assets/2179105/8aecdcf2-8c8d-4f89-9c55-174bdd5ca6b3)

Reply:
Perfect! Thanks for the info.